Makefile

```diff
@@ -9,6 +9,8 @@ submodules: ## Download the git submodules
 test: ## Execute the node tests
 	@echo "Running the tests"
 	@go test ./...
+	@echo "Running the end-to-end tests"
+	@go test -count=1 ./test --tags=endtoendtests
```

Comment:
It would be nice to have three different targets: one for unit tests, one for e2e tests, and one that runs both.

Reply:
I though about at first! OK!

Reply:
Done at [2197aa7](https://github.com/cartesi/rollups-node/pull/381/commits/2197aa76bbca18db6853792a8f952ddc4e15cac1)